toptal/picasso
# feat(input): tweak TextArea counter position

[FX-690](https://toptal-core.atlassian.net/browse/FX-690)

### Description

Describe the changes and motivations for the pull request, unless evident from the title.

### How to test

- Make sure the single line input has not changed at all
- Check updated styling and copy changes of TextAreas with limits - https://picasso.toptal.net/fx-690-textarea-counter-position/?path=/story/forms-folder--input#with-limit

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| ![image](https://user-images.githubusercontent.com/12392666/72339622-8c814b00-3701-11ea-97c9-e6889e37807f.png) | ![image](https://user-images.githubusercontent.com/12392666/72339431-2bf20e00-3701-11ea-8cbd-923c5fc02626.png)  |

### Review
